genie-web/build.gradle

```diff
@@ -117,6 +117,8 @@ node {
 }
 
 task bundle(type: NpmTask) {
+    inputs.files(fileTree("${project.projectDir}/src/main/resources/static"))
```

Comment:
why not just use `inputs.dir` instead of `inputs.files` and creating a file tree manually?